app/src/main/java/org/mozilla/rocket/shopping/search/ui/ShoppingSearchResultTabFragment.kt

```diff
@@ -251,6 +252,7 @@ class ShoppingSearchResultTabFragment : Fragment(), ContentTabViewContract, Back
                 override fun onPageScrolled(position: Int, positionOffset: Float, positionOffsetPixels: Int) = Unit
 
                 override fun onPageSelected(position: Int) {
+                    tab_layout_scroll_view.scrollTo(calculateScrollXForTab(position, 1F), 0)
```

Comment:
should we use smooth scrolling?

Reply:
Yes, I've added the scroll animation from TabLayout.